src/page-objects/base.ts

```diff
@@ -179,12 +179,13 @@ export default class BasePageObject {
   }
 
   async isDisplayed(selector: string) {
```

Comment:
Not sure if changing this method is safe, it has too many usages.

But if you checked this and it is safe, let's do that as well